test/python/compiler/test_assembler.py

```diff
@@ -93,6 +95,27 @@ def test_assemble_initialize(self):
         np.testing.assert_almost_equal(qobj.experiments[0].instructions[0].params,
                                        [0.7071067811865, 0, 0, 0.707106781186])
 
+    def test_assemble_single_schedule(self):
+        """Test assembling a single schedule.
+        """
+        schedule = ReferenceSchedules.nonsense()
+        config = {
+            'shots': 2000,
+            'qubit_lo_freq': [],
+            'meas_lo_freq': [],
+            'meas_level': 1,
+            'meas_return': 'avg',
+            'memory_slot_size': 100,
+            'rep_time': 1000
+        }
+        qobj = assemble_schedules(pulse.schedule.ConfiguredSchedule(schedule),
+                                  dict_header={},
+                                  dict_config=config)
+        self.assertIsInstance(qobj, PulseQobj)
+        self.assertEqual(qobj.config.shots, 2000)
+        self.assertEqual(len(qobj.experiments), 1)
+        self.assertEqual(qobj.experiments[0].instructions[0].name, 'pulse0')
+
```

Comment:
You need to check qobj itself. You can directly call `PulseQobj` and generate a reference.

Reply:
I think this is enough so far. Test for circuits do the same things like this. You can do this later if you want. Can we leave this as is?